bindings/rust/standard/certs/cert.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't think these certs need to be duplicated. I think the certs directory should probably just be in bindings/rust

Reply:
Moved to symlinks, since we're already doing this for `tests/pems`